src/components/projects/Header.svelte

```diff
@@ -18,12 +18,10 @@
     width: 100%;
   }
   h1 {
-    font-size: 64px;
-    font-family: 'Visby CF', sans-serif;
     color: white;
   }
   p {
-    font-size: 24px;
+    font-size: var(--body-xl);
     color: white;
```

Comment:
create a variable for white?

Reply:
Haha, that's pretty over-engineered. Let's stick with white.
That said, I do want to have a variable for black, since we actually use `#333` for an off-black (looks nicer to the eyes)

Reply:
Just to put in my 2&cent;, I would agree with James here. `var(--text-light)` or `var(--text-secondary)` or just `var(--secondary)` maybe?